sycl/test-e2e/Graph/Inputs/buffer_fill.cpp

```diff
@@ -0,0 +1,88 @@
+// Tests adding a Buffer fill operation as a graph node.
+
+#include "../graph_common.hpp"
+
+int main() {
+
+  queue Queue{{sycl::ext::intel::property::queue::no_immediate_command_list{}}};
+
+  if (!are_graphs_supported(Queue)) {
+    return 0;
+  }
```

Comment:
Unrelated to this PR, but I think the support should be printed as part of `sycl-ls` and then handled via `REQUIRES` to properly mark skipped tests.

Reply:
That's a good idea we hadn't considered, I've created an internal ticket for our team to do this as a follow-up standalone PR for all tests.